### ISSUE TITLE
Inconsistent error from `createUser` promise

### ISSUE THREAD

**Issue by grifotv** [auth0/react-native-auth0#25]
When running from iOS Simulator, I get one of the following errors in `error.name`:

- `user_exists` = email already in use
- `username_exists` = username already in use
- `invalid_password` = it includes a detailed explanation in `error.message.verified` and `error.message.rules`

However, when running on the device, I always get the same error:

```
TypeError: Error.captureStackTrace is not a function (In 'Error.captureStackTrace(_this,_this.constructor)', 'Error.captureStackTrace' is undefined)
```

![image uploaded from ios](https://cloud.githubusercontent.com/assets/435351/24451407/9392f890-1476-11e7-8aab-947d0da0d56b.jpg)

**Comment by grifotv** [comment]
I can confirm I get this same `TypeError: Error.captureStackTrace...` when running from iOS Simulator with the debugger window (http://localhost:8081/debugger-ui) on Safari.

<img width="501" alt="screen shot 2017-03-29 at 12 16 23" src="https://cloud.githubusercontent.com/assets/435351/24452108/92e30b26-1479-11e7-8dcd-df4d101ffb76.png">


If I check the debugger window on Chrome it works fine.

**Comment by grifotv** [comment]
I guess it might be related to this comment https://github.com/facebook/react-native/issues/7421#issuecomment-217396783

**Comment by hzalaz** [comment]
Closed by #33
